### ISSUE TITLE
ICD10 question

### ISSUE THREAD

**Issue by igerne94** [Conteir/HAPI-search-with-EHR-simulation#1]
The response from ICD-10 is always empty

**Comment by eirikconteir** [comment]
Yes it is. And the reason is (besides from the fact that very little content is actually tagged with an ICD-10 code):

Hapi don't allowed "." in the code. Eg: Covid-19, U07.1, is expressed as U071 (without full stop).

This is very easy to trim on our side, but I'm not sure we should do so. I suggest you:

1. talk briefly with Hilde about the problem 
2. do some Google investigation: Is it common to express ICD-10 codes without the dot/full stop in a machine-to-machine interacting on? 
3. ask Audun about his view... Is the full stop/dot an issue in API context?
4. If 2 and 3 returns false, you should again speak to Hilde, and ask her to make the necessary changes in HAPI
